go/vt/vttablet/onlineddl/executor.go

```diff
@@ -2041,6 +2070,7 @@ func (e *Executor) failMigration(ctx context.Context, onlineDDL *schema.OnlineDD
 	if err != nil {
 		_ = e.updateMigrationMessage(ctx, onlineDDL.UUID, err.Error())
 	}
+	e.ownedRunningMigrations.Delete(onlineDDL.UUID)
```

Comment:
This is a bugfix. The bug was exposed by this PR and was applicable for failed `online` migrations, which is why `onlineddl_vrepl` kept failing.
In `failMigration`, we need to remove the migration from the list of known, running migrations. Otherwise, the scheduler will avoid retrying the migration.